.github/workflows/build.yml

```diff
@@ -158,3 +158,39 @@ jobs:
       env:
         TESTS: ${{ matrix.suite }}
       run: ./ci/ci-script.bash
+
+  attributes-check:
+    needs: build
+    runs-on: ubuntu-20.04
+    name: Ubuntu 20.04 | Attributes check
```

Comment:
Please use 22.04 as it will last longer and that is where I do development.

However can we instead just run this as a test_regress/t/t_dist_attributes.pl as part of normal regressions (when on 22.04)?   That saves a whole checkout and works better for my personal flow.

Reply:
> Please use 22.04 as it will last longer and that is where I do development.

At first, I've used 22.04, but it turned out that verilator is only build for 20.04 in PRs.
I need source code after build as some parts of the files are generated using ``astgen``.
 
> However can we instead just run this as a test_regress/t/t_dist_attributes.pl as part of normal regressions (when on 22.04)? That saves a whole checkout and works better for my personal flow.

Sure, I will try to do this.

Reply:
I switched to ``test_regress`` format, but I'm unsure how to print test output in the CI without failing the test and re-running it. Test takes about 6-7 minutes to check all files and it seems like waste of CI time to re-run it.

There are some unsafe calls in verilator, so for now, I decided to create golden file and compare result to the golden file, but in the future we probably would like to fail test when we detect any unsafe call.

It should be ready for review.